app/uk/gov/hmrc/cgtpropertydisposals/models/SubscriptionDisplayResponse.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposals.models
+
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.cgtpropertydisposals.models.address.Address
+import uk.gov.hmrc.cgtpropertydisposals.models.ids.CgtReference
+import uk.gov.hmrc.cgtpropertydisposals.models.name.ContactName
+
+final case class SubscriptionDisplayResponse(
+  emailAddress: Option[Email], //TODO: DES have this as an optional field. Discuss.
```

Comment:
I don't think this should be optional but in the DES schema this is not mandatory

Reply:
it's officially an optional field in the spec but we will always send one over in subscription create so we can always expect one back.

Reply:
I don't know whether we want to be able to test when we don't get an email address though

Reply:
did we not have a discussion with them on this so they enforce the correct semantics ie we will always send this so make it non-optional